src/js/behaviors/fix-list-width.js

```diff
@@ -8,15 +8,27 @@ export default Behavior.extend({
 
     this.listenTo(this.view, 'fix:list:width', this.fixWidth);
   },
+  ui: {
+    listHeader: '.js-list-header',
+    list: '.js-list',
+  },
+  onChildViewAttach() {
+    this.fixWidth();
+  },
+  onChildViewRenderChildren() {
+    if (!this.view.isAttached()) return;
```

Comment:
Is this correct? Or should it be a check if the child view attached?

Reply:
The children are rendered the first time prior to dom attachment when width measurements can't be made. Really though the is attached check can just replace the is rendered check in fixWidth.

Reply:
Okay makes sense, I updated it to that.